### DISCUSSION TITLE
Send Message to number that not in contacts

### DISCUSSION THREAD

**Discussion by luridarmawan** [vasani-arpit/WBOT#287] [General]
Hi @vasani-arpit 

How to send message to number that not in contacts?
It should be save number to contacts first, then send the message.

Thank you.

**Comment by vasani-arpit** [comment]
The bot is developed in a way that it will only reply to messages but underlying lib(wapi.js) supports all sort of things.

With few modifications, you can send messages to the person which is not saved as contact.

Be aware that WA might ban you if you do so.

**Reply by luridarmawan** [reply]
yes, understood.
